mmpose/apis/inference_tracking.py

```diff
@@ -80,21 +181,45 @@ def get_track_id(results, results_last, next_id, iou_thr=0.3):
                 last frame (bbox_result, pose_result, track_id).
         next_id (int): The track id for the new person instance.
         iou_thr (float): The threshold for iou tracking.
+        oks_thr (float): The threshold for oks tracking.
+        min_keypoints (int): Minimum number of keypoints recognized as person.
+                            default:3
+        use_oks (bool): Flag to using oks tracking. default:False
+        use_one_euro (bool): Option to use one-euro-filter. default:False.
+        fps (optional): Parameters that d_cutoff
+                        when one-euro-filter is used as a video input
 
     Returns:
         list[dict]: The bbox & pose & track_id info of the
                 current frame (bbox_result, pose_result, track_id).
         int: The track id for the new person instance.
     """
+    results = _get_area(results)
 
-    for result in results:
-        track_id, results_last = _track_by_iou(result, results_last, iou_thr)
+    if use_oks:
+        _track = _track_by_oks
+        thr = oks_thr
+    else:
+        _track = _track_by_iou
+        thr = iou_thr
 
+    for result in results:
+        track_id, results_last, match_result = _track(result, results_last,
+                                                      thr)
         if track_id == -1:
-            result['track_id'] = next_id
-            next_id += 1
+            if np.count_nonzero(result['keypoints'][:, 1]) > min_keypoints:
+                result['track_id'] = next_id
+                next_id += 1
+            else:
+                result['keypoints'][:, 1] = -10
```

Comment:
Why it is -10 here? Please consider adding a few descriptions.

Reply:
In the case of bottom-ups, there are many instances of people with two to three keypoints where there are no real people.
Change the y-value of a human instance with a smaller number of keypoints than min_keypoints to -10 so that it does not appear on the screen. If you change to zero, a semicircle appears at the top of the screen.